pandora-server-directory/src/config.ts

```diff
@@ -1,102 +1,112 @@
-export const APP_NAME = process.env.APP_NAME ?? 'Pandora Directory Server';
+import { CreateEnvParser } from 'pandora-common';
+import { z } from 'zod';
 
-//#region Networking
+export const EnvParser = CreateEnvParser({
 
-/** Port on which will HTTP server listen */
-export const SERVER_PORT = process.env.SERVER_PORT ?? '25560';
-/** Path to HTTPS certificate file */
-export const SERVER_HTTPS_CERT = process.env.SERVER_HTTPS_CERT ?? '';
-/** Path to HTTPS key file */
-export const SERVER_HTTPS_KEY = process.env.SERVER_HTTPS_KEY ?? '';
-/** Secret key used to authenticate Shards connecting to Directory */
-export const SHARD_SHARED_SECRET = process.env.SHARD_SHARED_SECRET ?? '';
-/** How many hops are we after a trusted reverse proxy */
-export const TRUSTED_REVERSE_PROXY_HOPS = Number.parseInt(process.env.TRUSTED_REVERSE_PROXY_HOPS ?? '0') || 0;
+	APP_NAME: z.string().default('Pandora Directory Server'),
 
-//#endregion
+	//#region Networking
 
-//#region Logging
+	/** Port on which will HTTP server listen */
+	SERVER_PORT: z.number().default(25560),
```

Comment:
Nit: This should also specify it should be int, to preserve the removed check
```suggestion
	SERVER_PORT: z.number().int().default(25560),
```